pulsar-common/src/main/java/org/apache/pulsar/common/conf/InternalConfigurationData.java

```diff
@@ -28,19 +28,19 @@
 
     private String zookeeperServers;
     private String configurationStoreServers;
-    private String ledgersRootPath;
```

Comment:
for backward compatibility, you shouldn't change the field directly. 

You can add a new field `bookkeeperMetadataServiceUri` and deprecate the old field `ledgersRootPath `.

Reply:
Thanks for pointing this out, done.